src/app/products/floating-rates/create-floating-rate/create-floating-rate.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="floatingRateForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="row wrap" fxLayout.lt-md="column" fxLayoutGap="4%">
+
+          <mat-form-field fxFlex="30%">
+            <mat-label>Floating Rate Name</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="floatingRateForm.controls.name.hasError('required')">
+              Floating Rate Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <div class="checkbox-wrapper" fxFlex="30%">
+            <mat-checkbox formControlName="isBaseLendingRate" class="checkbox" labelPosition="before">Is Base Lending
+              Rate?</mat-checkbox>
+          </div>
+
+          <div class="checkbox-wrapper" fxFlex="30%">
+            <mat-checkbox formControlName="isActive" class="checkbox" labelPosition="before">Active</mat-checkbox>
+          </div>
+
+        </div>
+
+        <br>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <br>
+
+        <div fxLayout="row wrap" fxLayoutGap="60%" fxLayout.lt-md="column">
+
+          <p fxFlex="20%" class="mat-title">Floating Rate Periods</p>
+
+          <button mat-raised-button fxFlex="20%" type="button" color="primary" (click)="addFloatingRatePeriod()">
+            <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+            Add Floating Rate Period
```

Comment:
Since button is besides the label `Floating Rate Period` only plus sign is enough.